tvbwidgets/ui/spacetime_widget.py

```diff
@@ -0,0 +1,89 @@
+import numpy as np
+import pythreejs as p3
+import matplotlib.pyplot as plt
+import matplotlib.colors as mcolors
+from IPython.display import display
+from tvb.datatypes.connectivity import Connectivity
+from tvbwidgets.ui.base_widget import TVBWidget
+
+class SpaceTimeVisualizerWidget(TVBWidget):
+    def __init__(self, connectivity, width=600, height=400, **kwargs):
+        style = self.DEFAULT_BORDER
+        super().__init__(**kwargs, layout=style)
+        self.view_width = width
+        self.view_height = height
+        self.connectivity = connectivity
+        self.from_time = 0.00
+        self.to_time = 153.47
+        self.conduction_speed = 1.0
```

Comment:
I think in the end these should be retrieved programmatically and not be hardcoded.

Reply:
Hi, I’ve added these options—could you take a look and let me know if any improvements are needed?